sync2/handler2/handler.go

```diff
@@ -546,7 +550,7 @@ func (h *Handler) EnsurePolling(p *pubsub.V3EnsurePolling) {
 	log := logger.With().Str("user_id", p.UserID).Str("device_id", p.DeviceID).Logger()
 	log.Info().Msg("EnsurePolling: new request")
 	defer func() {
-		log.Info().Msg("EnsurePolling: request finished")
+		log.Info().Msg("EnsurePolling: preprocessing done")
```

Comment:
I know this isn't a traditional request/response, but the log line as it was written confused me.